Automattic/pocket-casts-android
# Transcripts - Update html format transcript display

## Description
This fixes a few html format transcript display as follows:
1. Loads them in web view if they contain javascript
2. Replace apostrophe and quote using regx

Internal Ref: p1726671990578069-slack-C078746V6UX

## Testing Instructions
1. Install the app from this branch
2. Open the feeds mentioned below
3. ✅ Notice that the corresponding pages are loaded as described 

## Screenshots or Screencast 
----

I've had it
feed: https://www.podcastone.com/podcast?categoryID2=2332
Loads page in webview, search icon is hidden

<img width=200 src="https://github.com/user-attachments/assets/f4c97e86-deab-4040-9108-f32f2c0a96d9"/>

----

TWit 
feed: https://feeds.twit.tv/sn.xml
This opens the HTML in the browser. 

<img width=200 src="https://github.com/user-attachments/assets/48322f31-ad43-462c-99b4-f97e7a691f4d"/>

----

Brain Science
feed: https://changelog.com/brainscience/feed
Replaces `&#39;` and `&quot;` with apostrophe and ".

<img width=200 src="https://github.com/user-attachments/assets/489b9240-0f70-4ede-aa86-d3dcf9cddc30"/>

----

If books could kill 
feed: https://feeds.buzzsprout.com/2040953.rss
Continue to be shown as before.

<img width=200 src="https://github.com/user-attachments/assets/380fc8ed-c719-4b44-94a9-e362dec87735"/>


## Checklist
- [ ] If this is a user-facing change, I have added an entry in CHANGELOG.md [To be added]
 
#### I have tested any UI changes...
